app/controllers/admin/auction_archive_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Admin::AuctionArchiveController < Admin::BaseController
+  def update
```

Comment:
thoughts on using the regular `admin/auctions#update` controller instead? I did that in my recent refactor PR and found it was helpful to reuse that rather than having a separate controller